runtime/src/accounts_db.rs

```diff
@@ -5568,6 +5627,25 @@ impl AccountsDb {
                 Some(&flushed_store),
                 None,
             );
+
+            if filler_accounts > 0 {
+                // add extra filler accounts at the end of the append vec
+                let (account, hash) = self.get_filler_account(&Rent::default());
```

Comment:
Hmm, is there a way to get the real `Rent` here? I think the current version is able to use the actual `Rent` instead of the default.

Reply:
I didn't see an easy way. These are fake accounts for stress testing - they just need to pass the rent check in bank's rent collection. Default is sufficient atm.